projects/kit/components/breadcrumbs/breadcrumbs.module.ts

```diff
@@ -1,28 +1,12 @@
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
-import {RouterModule} from '@angular/router';
-import {
-    TuiItemDirective,
-    TuiItemModule,
-    TuiLetModule,
-    TuiMapperPipeModule,
-} from '@taiga-ui/cdk';
-import {TuiLinkModule, TuiModeModule, TuiSvgModule} from '@taiga-ui/core';
+import {TuiSvgModule} from '@taiga-ui/core';
 
 import {TuiBreadcrumbsComponent} from './breadcrumbs.component';
 
 @NgModule({
-    imports: [
-        CommonModule,
-        RouterModule,
-        TuiMapperPipeModule,
-        TuiLetModule,
-        TuiLinkModule,
-        TuiSvgModule,
-        TuiModeModule,
-        TuiItemModule,
-    ],
+    imports: [CommonModule, TuiSvgModule],
     declarations: [TuiBreadcrumbsComponent],
-    exports: [TuiBreadcrumbsComponent, TuiItemDirective],
```

Comment:
no, dont delete it)

The idea of this export - user should not worry about importing of `TuiItemModule`

Reply:
you are right)